pages/item/_itemId.vue

```diff
@@ -11,6 +11,29 @@
         <v-btn>{{ $t('more') }}</v-btn>
       </v-col>
     </v-row>
+    <div v-if="item.Type === 'Series'">
+      <v-tabs v-model="seasonTabs">
+        <v-tab v-for="season in allEpisodes" :key="season.Id">
+          {{ season.Name }}
+        </v-tab>
+      </v-tabs>
+
+      <v-tabs-items v-model="seasonTabs">
+        <v-tab-item v-for="season in allEpisodes" :key="season.Id">
+          <v-slide-group show-arrows="mobile">
+            <v-slide-item v-for="episode in season.Episodes" :key="episode.Id">
+              <v-card class="ma-5">
+                <v-img :src="getImageLink(episode.Id, 'primary')"></v-img>
+                <v-card-subtitle>
+                  <span>Episode {{ episode.IndexNumber }}</span>
```

Comment:
This is an excellent time to switch to SXXEYY if we want to, and I'd be happy for the change.

Reply:
I think "Episode XYZ" look cleaner if we go for this kind of layout.

But this definitely needs a translated string (And also remove the `span`)

Reply:
why remove the span ;-;

Reply:
There is the season number just above it. So "episode X" looks better to me.
I'll make it a computed property and add the string

Reply:
> why remove the span ;-;

It makes templates more dirty and harder to read, for basically no benefit.